README.rst

```diff
@@ -32,16 +34,25 @@ Free software: Apache License v2
 All images contained in any (sub-)directory of this repository are licensed under the CC0 license which can be found
 `here <https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt>`__.
 
-Feature overview
-----------------
+Major ISOFIT 3x features
+------------------------
+
+* new handling of look-up-tables (LUTs), including the option to provide custom prebuilt LUTs
+* click command line utilities, including download of external data and example files
+* a more flexible isofit.ini file used to discover various paths such as tests, data, and examples
+* instructions for dev environment setup and a collection of setup scripts
+* numpy implementation of the sRTMnet emulator (removes tensorflow dependency)
```

Comment:
Let's add:
- "centralized radiative transfer physics for more flexible development and experimentation"
- "test coverage for major functionality"